operation/operation.c

```diff
@@ -169,17 +169,25 @@ void print_state(oper_st state) {
            state.pills_detected);
 }
 
+// Prints stored EEPROM logs.
+void press_sw_to_read_log(oper_st * state) {
```

Comment:
can you also include this function in operation.h

Reply:
I can... Is it needed elsewhere?

Reply:
I think this point is one place where we can skim down our code: redundant .h definitions ; not everything needs to be shared, imo

Reply:
I wouldnt consider it as sharing between multiple source files, but more as function declarations in header, potentially can affect compiling speed.